tests/js/read-write.test.ts

```diff
@@ -89,3 +89,21 @@ it("reads empty file", async (t) => {
   expect(table.numCols).toStrictEqual(0);
   // console.log("empty table schema", table.schema);
 });
+
+it("read stream-write stream-read stream round trip (no writer properties provided)", async (t) => {
+  const server = await temporaryServer();
+  const listeningPort = server.addresses()[0].port;
+  const rootUrl = `http://localhost:${listeningPort}`;
+
+  const expectedTable = readExpectedArrowData();
+
+  const url = `${rootUrl}/1-partition-brotli.parquet`;
+  const originalStream = await wasm.readParquetStream(url);
+
+  const stream = await wasm.transformParquetStream(originalStream);
+  const accumulatedBuffer = new Uint8Array(await new Response(stream).arrayBuffer());
```

Comment:
Out of curiosity, how would you write this to a file in Node? Can you pass the `stream` object to a node file?

Reply:
It turns out to be quite ergonomic:
```javascript
const destinationWritable = Writable.toWeb(handle.createWriteStream());
await outputStream.pipeTo(destinationWritable);
```
alternatively:
```javascript
await handle.writeFile(outputStream)
```

Deno's version of the former is pretty succinct too:
```javascript
await outputStream.pipeTo(handle.writable);
```

(there's ~10 different ways to do it (e.g. `fs.writeFile(path, inputStream)` is just a shortcut for explicitly creating a stream.Writable and piping to it), these strike a reasonable balance).